core/spec/models/spree/shipping_rate_spec.rb

```diff
@@ -11,20 +11,20 @@
 
   context "#display_price" do
     context "when tax included in price" do
+      let!(:default_zone) { create(:zone, :default_tax => true) }
+      let(:default_tax_rate) do
+        create(:tax_rate,
+          :name => "VAT",
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.